yt_dlp/extractor/aenetworks.py

```diff
@@ -73,18 +73,21 @@ def _extract_aen_smil(self, smil_url, video_id, auth=None):
 
     def _extract_aetn_info(self, domain, filter_key, filter_value, url):
         requestor_id, brand, software_statement = self._DOMAIN_MAP[domain]
+        graphql_video_id = (self._search_regex(
+            r'<meta[^>]+content="[^"]*tpid/(\d+)"|<meta[^>]*name="videoId"[^>]*content="(\d+)"',
```

Comment:
Simpler? But also consider if you could use `_html_search_meta()`.
```suggestion
            r'<meta\s[^>]*(?:content="[^"]*tpid/(\d+)"|name="videoId"[^>]+content="(\d+)")',
```
THe proposed pattern works because of a little-known feature (that I don't remember seeing used before) of `_search_regex()` where the first matched group if no group(s) is specified.